card/mod/01_core/set/all/utils.rb

```diff
@@ -2,11 +2,11 @@
 module ClassMethods
   
   def empty_trash
+    Card.delete_trashed_files
```

Comment:
this looks through all the files associated with trashed cards and deletes them.  won't work if we first destroy all the trashed cards!

(certainly might be better to find files not associated with any cards, but this is enough to get it so that the bootstrap stuff doesn't create unwanted file directories)